ebpf-for-windows.sln

```diff
@@ -149,6 +150,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "nuget", "tools\nuget\nuget.
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "pe-parse", "libs\pe-parse\pe-parse.vcxproj", "{FE4FEA79-BFBB-4822-ABCB-0D3BEEA240A7}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "spec", "libs\spec\spec.vcxproj", "{C3D2CD73-BF4C-47DF-8808-2A9996124D5B}"
```

Comment:
"spec" sounds like it should contain the eBPF spec.  My understanding is it's about ELF not eBPF per se.  Maybe rename to elf-spec